com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -877,8 +870,14 @@ public void Shutdown()
                 NetworkTickSystem.Tick -= OnNetworkManagerTick;
                 NetworkTickSystem = null;
             }
-
-            IsListening = false;
+            // This is required for handling the potential scenario where multiple NetworkManager instances are created.
+            // See MTT-860 for more information
+#if !UNITY_2020_2_OR_NEWER
```

Comment:
we no longer target below 2020.3 therefore I don't think if-guard is needed here.

Reply:
Thank You!
That did get left in and will be removed shortly.